src/coreclr/System.Private.CoreLib/src/System/Exception.CoreCLR.cs

```diff
@@ -274,5 +274,32 @@ private bool CanSetRemoteStackTrace()
 
             return true;
         }
+
+        // used by vm
+        internal string? GetHelpContext(out uint helpContext)
+        {
+            helpContext = 0;
+            string? helpFile = HelpLink;
+
+            int poundPos, digitEnd;
+
+            if (helpFile is null || (poundPos = helpFile.LastIndexOf('#')) == -1)
+            {
+                return helpFile;
+            }
+
+            for (digitEnd = poundPos + 1; digitEnd < helpFile.Length; digitEnd++)
+            {
+                if (char.IsWhiteSpace(helpFile[digitEnd]))
+                    break;
+            }
+
+            if (uint.TryParse(helpFile.AsSpan(poundPos + 1, digitEnd - poundPos - 1), out helpContext))
```

Comment:
Should this not be culture invariant?

Reply:
AFAIK the default format of uint32 isn't affected by culture at all. Negative sign, digit separator and decimal point are not allowed.